### PR TITLE
Add google analytics

### PR DESCRIPTION
**Pull request by memnoth**
Add google analytics

### CODE REVIEW

**Comment by rsangole** [comment]
This is a useful feature.

**Comment by jamestharpe** [comment]
I think a better approach would be to support includes at various points in the template. Then users can copypasta any analytics snippets they want. I'll try to submit a pull request with this approach shortly.

**Comment by jamestharpe** [comment]
New PR opened here: https://github.com/calintat/minimal/pull/28

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -3,6 +3,16 @@
             <p class="copyright text-muted">{{ .Site.Params.copyright | default "&copy; All rights reserved. Powered by [Hugo](https://gohugo.io) and [Minimal](https://github.com/calintat/minimal)" | markdownify }}</p>
 
         </footer>
+        <!-- Global Site Tag (gtag.js) - Google Analytics -->
```
Comment (jamestharpe) [inline layouts/partials/footer.html]: Per Google's documentation, this code should go just after the opening <head> tag.

Comment (memnoth) [reply layouts/partials/footer.html]: Thanks for thata
For further detail, https://groups.google.com/d/msg/digital-analytics-fundamentals/V9z4TVXQW_4/OgW6M9xJLqAJ

**Pull request merged**